data/i18n/qml_he.po

```diff
@@ -0,0 +1,560 @@
+msgctxt "main|"
+msgid "Welcome"
+msgstr "ברוך בואך"
+
+#: qrc/main.qml:131
+msgctxt "main|"
+msgid "Ahoy, this is openSUSE"
+msgstr "אהוי, זוהי openSUSE"
+
+#: qrc/main.qml:134
+msgctxt "main|"
+msgid "Basics"
+msgstr "הבסיס"
+
+#: qrc/main.qml:135
+msgctxt "main|"
+msgid "Support"
+msgstr "תמיכה"
+
+#: qrc/main.qml:138
+msgctxt "main|"
+msgid "Read me"
+msgstr "קראו אותי"
+
+#: qrc/main.qml:139
+msgctxt "main|"
+msgid "Documentation"
+msgstr "תיעוד"
+
+#: qrc/main.qml:140
+msgctxt "main|"
+msgid "Get Software"
+msgstr "קבלת תוכנה"
+
+#: qrc/main.qml:143
+msgctxt "main|"
+msgid "Install openSUSE"
+msgstr "התקנת openSUSE"
+
+#: qrc/main.qml:146
+msgctxt "main|"
+msgid "GNOME Help"
+msgstr "GNOME עזרה"
+
+#: qrc/main.qml:147
+msgctxt "main|"
+msgid "Plasma Help"
+msgstr "עזרה ב־Plasma"
+
+#: qrc/main.qml:148
+msgctxt "main|"
+msgid "Xfce Help"
+msgstr "עזרה ב־Xfce"
+
+#: qrc/main.qml:149
+msgctxt "main|"
+msgid "MATE Help"
+msgstr "MATE עזרה"
+
+#: qrc/main.qml:150
+msgctxt "main|"
+msgid "Enlightenment Help"
+msgstr "עזרה ב־Enlightenment"
+
+#: qrc/main.qml:151
+msgctxt "main|"
+msgid "About Budgie"
+msgstr "על אודות Budgie"
+
+#: qrc/main.qml:152
+msgctxt "main|"
+msgid "About Lumina"
+msgstr "על אודות Lumina"
+
+#: qrc/main.qml:153
+msgctxt "main|"
+msgid "LXDE Help"
+msgstr "עזרה ב־LXDE"
+
+#: qrc/main.qml:154
+msgctxt "main|"
+msgid "LXQt Help"
+msgstr "עזרה ב־LXQt"
+
+#: qrc/main.qml:155
+msgctxt "main|"
+msgid "About Cinnamon"
+msgstr "על אודות Cinnamon"
+
+#: qrc/main.qml:156
+msgctxt "main|"
+msgid "Deepin Help"
+msgstr "עזרה ב־Deepin"
+
+#: qrc/main.qml:159 qrc/main.qml:235
+msgctxt "main|"
+msgid "Contribute"
+msgstr "התנדבות"
+
+#: qrc/main.qml:160
+msgctxt "main|"
+msgid "Build openSUSE"
+msgstr "בניית openSUSE"
+
+#: qrc/main.qml:163
+msgctxt "main|"
+msgid ""
+"If this is your first time using openSUSE, we would like you to feel right "
+"at home in your new voyage. Take your time to familiarize yourself with all "
+"the buttons and let us know how you like the experience on our"
+msgstr ""
+"אם זוהי הפעם שבה הנך משתמש/ת שלך ב־operSUSE, ברצוננו שתרגיש/י כמו בבית "
+"בהפלגה החדשה שלך. נא לקחת את הזמן להתאים את עצמך עם כל הכפתורים וליידע אותנו "
+"בחוויה שלך ב"
+
+#: qrc/main.qml:164
+msgctxt "main|"
+msgid "social media"
+msgstr "מדיה החברתית שלנו"
+
+#: qrc/main.qml:167
+msgctxt "main|"
+msgid "Show on next startup"
+msgstr "הצגה בהפעלה הבאה"
+
+#: qrc/main.qml:168 qrc/main.qml:175
+msgctxt "main|"
+msgid "Customise"
+msgstr "התאמה אישית"
+
+#: qrc/main.qml:169 qrc/main.qml:228 qrc/main.qml:303 qrc/main.qml:341
+msgctxt "main|"
+msgid "Close"
+msgstr "סגירה"
+
+#: qrc/main.qml:176
+msgctxt "main|"
+msgid "Desktop Layouts"
+msgstr "פריסות שולחן עבודה"
+
+#: qrc/main.qml:177
+msgctxt "main|"
+msgid ""
+"openSUSE Xfce offers a variety of customisation options, including the "
+"ability to choose a desktop layout that fits your workflow. You can choose "
+"one of these as a starting point and customise it further later."
+msgstr ""
```

Comment:
There are some strings missing, so I assume this PR is still a draft?

Reply:
Hello, this PR isn't a draft. I'll may translate more strings in the future, but not on this PR.

Reply:
That does not make sense. Either it's complete - or it's a draft. And as long as the translation is not completed I'd rather not merge it.

Reply:
Now I completed the translation of openSUSE Welcome, and will open a new pull request with the full translation.